README.md

```diff
@@ -59,6 +59,38 @@ Objectives:
 - Extensible: Customizable without touching the core code.
 - Unified: Single codebase, deployable as an agent or collector with support for traces, metrics and logs.
 
+## Community
+
+The OpenTelemetry Collector SIG is present at the [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W)
+channel on the CNCF Slack and [meets once a week](https://github.com/open-telemetry/community#implementation-sigs) via
+video calls. Everyone is invited to join those calls, which typically serves the following purposes:
+
+- meet the humans behind the project
+- get an opinion about specific proposals
+- look for a sponsor for a proposed component after trying already via GitHub and Slack
+- get attention to a specific pull-request that got stuck and is difficult to discuss asynchronously
+
+Between 11 July 2024 and 09 January 2025, we'll have our video calls rotating between three time slots, in order to
+allow everyone to join at least once every three meetings:
+
+- [00:00 UTC](https://dateful.com/convert/utc?t=0000)
+- [12:00 UTC](https://dateful.com/convert/utc?t=1200)
+- [16:00 UTC](https://dateful.com/convert/utc?t=1600)
+
+Contributors to the project are also welcome to have ad-hoc meetings for synchronous discussions about specific points.
+Here is a list of tools that can be used for this:
```

Comment:
I'd suggest the list of tools here is unnecessary and would rather call out how one setup an ad-hoc meeting (ie. post a note in slack that says hey i want to meet to chat about X)